scripts/deployment/hf_upload/prepare.py

```
@@ -0,0 +1,199 @@
+        f.write(model_card)
+
+    # requirements.txt from template
+    req_path = os.path.join(templates_dir, 'requirements_model.txt.tmpl')
+    with open(req_path, 'r') as f:
+        requirements = f.read()
+    with open(os.path.join(temp_dir, 'requirements.txt'), 'w') as f:
+        f.write(requirements)
+
+    # Validate critical files
+    critical_files = ['config.json', 'tokenizer.json', 'tokenizer_config.json']
+    missing_files: List[str] = []
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)